Dockerfile

```
@@ -1,27 +1,47 @@
-FROM python:3.8-slim
-
-RUN apt-get update && apt-get -y install git gcc
-
-RUN useradd -r app_user
-RUN mkdir /app
-RUN chown app_user /app/
-WORKDIR /app
+FROM python:3.10-alpine
+
+ENV PYTHONPATH="/usr/local/src/idunn/src"
+ENV PYTHONUNBUFFERED=1
+
+# create the user idunn
+RUN mkdir /usr/local/src \
+ && addgroup --gid 1000 idunn \
+ && adduser \
+      --disabled-password \
+      --home /usr/local/src/idunn \
+      --ingroup idunn \
+      --shell /bin/sh \
+      --uid 1000 \
+      idunn
+
+RUN apk update \
+    && apk add --upgrade --no-cache \
+        bash openssh curl ca-certificates openssl less htop gcc git \
+		g++ make wget rsync \
+        build-base libpng-dev freetype-dev libexecinfo-dev openblas-dev libgomp lapack-dev \
+		libgcc libquadmath musl  \
+		libgfortran \
+		lapack-dev \
+	&&  pip install --no-cache-dir --upgrade pip
+
+WORKDIR /usr/local/src/idunn/src
 
 # Installing packages
-RUN pip install pipenv==2018.11.26
+RUN pip install pipenv
 
-ADD --chown=app_user app.py Pipfile* /app/
+ADD --chown=idunn app.py Pipfile* /usr/local/src/idunn/src/
```

Why did we need to change the path here?

On the other side I think this should be this:
```suggestion
ADD --chown=idunn app.py Pipfile* /usr/local/idunn/src/
```

---

Also, when trying to run the image *gunicorn* doesn't find the module Idunn, so it shoud probably even be something like this:
```suggestion
ADD --chown=idunn app.py Pipfile* /usr/local/idunn/
```

---

To be consinstend with Filesystem Hierarchy Standard (FHS) /app does not exist